### DISCUSSION TITLE
How can I disable RN's WebView from jumping to the browser?

### DISCUSSION THREAD

**Discussion by IveTian** [react-native-webview/react-native-webview#2164] [Q&A]
When I click on the link with `target="_blank"` in RN's WebView, it will jump to an external browser to continue. Is there any way to click on the link with `target="_blank"` and continue loading in the current RN WebView?

**Comment by nickap** [comment]
Hi @IveTian, 

I have the same issue!
Did you found any solution?
Thanks!

**Comment by putuoka** [comment]
add this prop on webview tag

    injectedJavaScriptBeforeContentLoaded={`
                    const links = document.querySelectorAll('a');                              
                    for (link of links) {
                        if (link.href.startsWith('mailto:')
                            || link.href.startsWith('tel:')
                            || link.href.startsWith('skype:')
                            || link.href.startsWith('whatsapp:')) {
                            link.target = '_blank'
                        } else {
                            link.removeAttribute('target');
                        }
                    }
                `}
